src/components/Colors/Colors.tsx

```diff
@@ -0,0 +1,60 @@
+import { Color } from "./Color";
+export function Colors() {
+  const colorBrand = "bg-[#1AE19D]";
+  const colorWhite = "bg-[#FFFFFF]";
+  const colorDarkGrey = "bg-[#DADADA]";
+  const colorLightGrey = "bg-[#F1F1F1]";
+  const colorDarkBlue = "bg-[#151A26]";
+  const colorBasicBlue = "bg-[#1A202E]";
+  const colorLightBlue = "bg-[#2E374C]";
+  const colorWarning = "bg-[#FFA41D]";
+  const colorError = "bg-[#FF4F4F]";
+  const light = [
+    { color: colorWhite, theme: "primary" },
```

Comment:
Themes are **light** or **dark** in our Tailwind world. I would suggest you to change that for **name** or **alias** or something else...

Reply:
this is outdated !

Reply:
Nice